### PR TITLE
fix: Use only major version to assign family for aws_db_parameter_group

### PR DESCRIPTION
**Pull request by FelisiaM**
### Checklist:

* [x] Have you added Draft Release Notes in `docs/draft-release-notes.md`?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

File: terraform/postgresql/provision/variables.tf [hunk 1]
```diff
@@ -20,8 +20,9 @@ variable "storage_gb" { type = number }
 variable "publicly_accessible" { type = bool }
 variable "multi_az" { type = bool }
 variable "instance_class" { type = string }
-variable "engine" { type = string }
-variable "engine_version" { type = string }
+variable "postgres_version" { type = string }
+#variable "postgres_version_major" { type = string }
```
Comment (jhvhs) [inline terraform/postgresql/provision/variables.tf:24]: *question:* Why do we want to keep these variables commented out?

Comment (FelisiaM) [reply terraform/postgresql/provision/variables.tf:24]: Nope- left there accidentaly

Comment (FelisiaM) [reply terraform/postgresql/provision/variables.tf:24]: Fixed it

Comment (blgm) [reply terraform/postgresql/provision/variables.tf:24]: I think pushing that change wasn't successful.

**Review by jhvhs — APPROVED** [review]

**Pull request merged**